README.md

```diff
@@ -42,6 +42,33 @@ var client = Serf.connect(function (err) {
 });
 ```
 
+All of them non-stream-based commands also have a promise interface if you'd
```

Comment:
Minor typo: s/them/the

Reply:
Oops my bad, will fix that now and add a test 👍